packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -312,6 +349,156 @@ export default Component.extend({
     return direction;
   },
 
+  /**
+   * Synchronizes the width of visible table headers with real (hidden) table header elements
+   *
+   * @method _syncHeadersWidth
+   * @private
+   * @returns {void}
+   */
+  _syncHeadersWidth() {
+    //in editing mode the real table headers are visible, no need to sync
+    if (get(this, 'isEditingMode')) {
+      return;
+    }
+
+    let widths = Array.from(this.element.querySelectorAll('th')).map(elm => elm.getBoundingClientRect().width);
+
+    let prevWidths = get(this, 'columnsWidth'),
+      widthChanged = widths.length !== prevWidths.length; //if number of columns has changed, set to true to re-set width
+
+    Array.from(this.element.querySelectorAll('div.table-header-row .table-header-cell')).map((headerElm, i) => {
+      let headerElmWidth = parseInt(headerElm.style.width);
+
+      if (!headerElmWidth || widths[i] !== prevWidths[i]) {
+        headerElm.style.width = `${widths[i]}px`;
+        widthChanged = true;
+      }
+    });
+
+    //set total width to headers wrapper div
+    let headerRowElm = this.element.querySelector('div.table-header-row'),
+      headerRowStyle = parseInt(headerRowElm.style.width);
+
+    if (!headerRowStyle || widthChanged) {
```

Comment:
what is the case that we need `parseInt`?

Reply:
`200px` for example